platform/beetle/inc/config.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2013-2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef __CONFIG_H__
+#define __CONFIG_H__
+
+/* uVisor stack configuration */
+#define STACK_SIZE       2048
```

Comment:
@fvincenzo These 2 lines could be removed. Could you please verify that it builds fine without them? (They should all have a default value).